app/src/main/java/com/baysoftware/bayfit/home/view/ExerciseSessionAdapter.kt

```diff
@@ -0,0 +1,46 @@
+package com.baysoftware.bayfit.home.view
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.baysoftware.bayfit.R
+import com.baysoftware.bayfit.db.ExerciseSessionEntity
+
+class ExerciseSessionAdapter(
+    private val objects: List<ExerciseSessionEntity>,
+    onClickListener: (Int) -> Unit
```

Comment:
modifique esse click listener para que ele envie um ExerciseItemEntity, ficará assim:
`private val onClickListener: (ExerciseSessionEntity) -> Unit`

isso faz com que quando o `onClickListener` for chamado, ele envie o `ExerciseSessionEntity` correspondente no evento de clique